isthmus/src/test/java/io/substrait/isthmus/TpcdsQueryNoValidation.java

```diff
@@ -42,9 +42,9 @@ private void testQuery(int i) throws Exception {
   @ParameterizedTest
   @ValueSource(
       ints = {
-        1, 3, 4, 6, 7, 8, 10, 11, 13, 14, 15, 16, 18, 19, 21, 22, 23, 25, 26, 28, 29, 30, 31, 32,
```

Comment:
seems you remove this line by accident.

Reply:
Sorry for the mistake, let me add this line.